content/api/seedlists.apib

```diff
@@ -0,0 +1,46 @@
+FORMAT: 1A
+title: Seed List API
+description: Retrieve your seed list to improve your deliverability.
+label: Deliverability
+
+# Group Seed List
+
+The Seed List API allows you to programmatically retrieve your seed list, which is used for measuring your deliverability.
+
+Note: The Seed List API is available to the SparkPost Deliverability Add-On customers only.
+ 
+### Retrieve Seed List [GET /v1/seeds]
+
+In order for a primary account to specify retrieval of a subaccount, the X-MSYS-SUBACCOUNT header must be set with the desired subaccount id. A primary or subaccount retrieving its own seed list does not need to explicitly set this.
+
+We recommend that you cache the seed list and update it when our Seed List Update Notification service sends out an email with the latest seed list.
```

Comment:
Is this also the guidance for API consumers (to react to an email, vs programmatically diff)?

Reply:
We decided to add this as a way to try and limit the amount of requests coming in as the seed list doesn't change super frequently. If someone is sending thousands of emails and grabbing that seedlist every time it would be better for them to cache it. Not sure if that answered your question?